python/ray/_private/accelerators/accelerator.py

```diff
@@ -86,7 +86,7 @@ def validate_resource_request_quantity(
     @staticmethod
     @abstractmethod
     def get_current_process_visible_accelerator_ids() -> Optional[List[str]]:
-        """Get the ids of accelerators of this family that are visible to the current process.
+        """Get the ids of accelerators of this family that are visible to the current process. # noqa:E501
```

Comment:
Can we break the line instead of ignoring the line-too-long error?

Reply:
Sure, I will handle that